app/helpers/template_helpers.py

```diff
@@ -75,10 +74,16 @@ def context(self) -> dict[str, Any]:
             "google_tag_manager_id": self._google_tag_manager_id,
             "google_tag_manager_auth": self._google_tag_manager_auth,
             "survey_type": self._survey_type,
+            "mastheadLogo": self._survey_config.masthead_logo,
+            "mastheadLogoMobile": self._survey_config.masthead_logo_mobile,
         }
         if self._survey_type:
             context["cookie_settings_url"] = self._survey_config.cookie_settings_url
             context["cookie_domain"] = self._survey_config.cookie_domain
+            if self._survey_title:
```

Comment:
Do we need this`if-else`? `survey_title` already has a value assigned in 
```
self._survey_title = cookie_session.get(
            "survey_title", self._survey_config.survey_title
        )
```
And passed into context:
```
"survey_title": self._survey_title,
```
if we need default value we should change it in `SurveyConfig` parent class?